dataactbroker/handlers/fileHandler.py

```diff
@@ -1268,10 +1270,12 @@ def list_submissions(page, limit, certified, sort='modified', order='desc', d2_s
 
     query = query.order_by(sort_order)
 
-    query = query.limit(limit).offset(offset)
+    submission_list = serialize_submission(query)
```

Comment:
This means we're querying for **all** submissions every time, which is very inefficient. This will cause issues when we have hundreds, then thousands of submissions.